api/facadeversions.go

```diff
@@ -109,7 +109,7 @@ var facadeVersions = map[string]int{
 	"SecretsManager":               1,
 	"Singular":                     2,
 	"Spaces":                       6,
-	"SSHClient":                    3,
+	"SSHClient":                    4,
```

Comment:
We'll need to update the minimum 2.9 version allowed to log into 3.0 after this lands and 2.9.35 is released

Reply:
Yeah.